docs/content/troubleshooting/troubleshoot-configmap-policy.md

```diff
@@ -0,0 +1,26 @@
+Checking the Events of a Policy Resource
+After you create or update a Policy resource, you can use kubectl describe to check whether or not the Ingress Controller accepted the Policy:
+
+$ kubectl describe pol webapp-policy
```

Comment:
```suggestion
`kubectl describe pol webapp-policy`
```

Reply:
we have `$` in all the commands in the docs...I'm down for removing it everywhere but probably we shouldn't have a mix of both right now?

Reply:
yea, I agree. Maybe remove the `$` so folks could copy paste without pulling that character in?